scripts/upgrade-test.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Runs a test to see what happens if we upgrade consensus
+
+set -euxo pipefail
+export RUST_LOG="${RUST_LOG:-info}"
+
+source ./scripts/setup-tests.sh
+
+server4=$(tail -4 $FM_PID_FILE | head -1)
+server3=$(tail -3 $FM_PID_FILE | head -1)
+server2=$(tail -2 $FM_PID_FILE | head -1)
+server1=$(tail -1 $FM_PID_FILE | head -1)
+
+
+function wait_server_shutdown() {
+  echo "Waiting for $1 to shutdown..."
+  tail --pid=$1 -f /dev/null
+  echo "Server $1 has shutdown."
+}
+
+mine_blocks 110
```

Comment:
FYI: we mine 101 blocks on bitcoind startup now so this might not be necessary https://github.com/fedimint/fedimint/blob/37a70ba4ef609a456bfb1323ca2a7b598f3b3d1b/fedimint-testing/src/bin/fixtures.rs#L161